thewalrus/quantum/distinguishable_squeezers.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2021-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Functions for calculating properties of states of distinguihable squeezed states of
+light having passed through an interferometer.
+"""
+
+import numpy as np
+from .photon_number_distributions import _squeezed_state_distribution
+
+
+def sample(T, rs, n_samples=100, input_cutoff=50):
+    """
+    Calculates a resultant photon number samples when distinguishable squeezers are sent into an
+    interferometer.
+
+    Args:
+        T (numpy.ndarray): interferometer transmission matrix
```

Comment:
hey @heltluke , I think the test should pass now. My only comment at this point is that elsewhere in TW we just write `array` instead `numpy.ndarray`. I take that likely the correct form to do it is the way you did it, but for consistency reasons I'd prefer to have everything in the same format.

Reply:
Done